### PR TITLE
[RHELC-1605] Add collecting of post-conversion report

### PR DESCRIPTION
**Pull request by hosekadam**
Add parsing of the post-conversion report is possible because in convert2rhel was added new feature enabling this.

Remember that post-conversion report is generated only if the analysis step is successful, otherwise only the pre-conversion report is generated.

[RHELC-1605](https://issues.redhat.com/browse/RHELC-1605)

### CODE REVIEW

File: roles/handle_c2r_report/tasks/check-results-file.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- name: Print the value of a variable with message
+  ansible.builtin.debug:
+    msg: "The value of my_variable is: {{ handle_c2r_report_result_filename_txt }}"
+- name: Set variable
+  ansible.builtin.set_fact:
+    result_file_exists: result.stat.exists
+- name: Check that result file exists
+  ansible.builtin.assert:
+    that: result.stat.exists
+    msg: "The analysis report file {{ handle_c2r_report_result_filename_txt }} was not created."
```
Comment (r0x0d) [inline roles/handle_c2r_report/tasks/check-results-file.yml]: This would need to be either analysis or conversion report, no?

**Review by r0x0d — COMMENTED** [review]
Just one small comment.

**Comment by djdanielsson** [comment]
Needs to pass the lint rules

File: roles/handle_c2r_report/tasks/check-results-file.yml (hunk 1 above)
Comment (hosekadam) [reply roles/handle_c2r_report/tasks/check-results-file.yml]: Yeah, the path of `handle_c2r_report_result_filename_txt` is being changed based on activity

**Comment by hosekadam** [comment]
@djdanielsson should be fine now. It just couldn't find the `parse_c2r_report` which I renamed to `handle_c2r_report`

File: roles/handle_c2r_report/tasks/check-results-file.yml (hunk 1 above)
Comment (r0x0d) [reply roles/handle_c2r_report/tasks/check-results-file.yml]: Yeah, but the `The analysis report file ` part...

**Review by r0x0d — APPROVED** [review]

File: roles/handle_c2r_report/tasks/check-results-file.yml (hunk 1 above)
Comment (hosekadam) [reply roles/handle_c2r_report/tasks/check-results-file.yml]: I see, I removed the `analysis` from the message.

**Review by r0x0d — APPROVED** [review]

**Pull request merged**